dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/SubmissionService.java

```diff
@@ -92,7 +92,7 @@
     public WorkspaceItem createWorkspaceItem(Context context, Request request) throws SQLException, AuthorizeException {
         WorkspaceItem wsi = null;
         Collection collection = null;
-        String collectionUUID = request.getHttpServletRequest().getParameter("collection");
+        String collectionUUID = request.getHttpServletRequest().getParameter("owningCollection");
```

Comment:
Will this affect the Angular UI submission process?  The REST Contract does say this param should be `owningCollection`, so this change is needed.  However, it seems like we may need a minor PR to the Angular UI to ensure this PR doesn't break submissions.

Reply:
There's no Angular UI using the collection or owningCollection parameter when creating a workspace item yet